calls-to-action.yaml

```diff
@@ -17,3 +17,18 @@
   Text: "Join Udi Dahan's in-depth Advanced Distributed Systems Design course"
   ButtonText: 'Learn more about the course'
   Url: 'https://particular.net/adsd'
+
+- Name: interest-apache-kafka
```

Comment:
I am unsure about the location of the callouts.
![image](https://github.com/Particular/docs.particular.net/assets/122651/a12f5a9a-a405-411b-a5f8-9b3c47c6d4c1)

I actually did not notice them.
It is probably just me, but I wonder if we would be able to gather more analytics from the existing callouts to figure out if they are working or not.

My personal opinion is that I would prefer the callouts to be in line with the content of the page; that way, the user is prompted for action as they are reading it.

In Octopus, the callouts to provide feedback are very prominent and positioned within where the user is focused.
![image](https://github.com/Particular/docs.particular.net/assets/122651/0c5fe150-935a-4149-b3be-4cbc8aded4da)
Hence, it makes them stand out, and that seems to work well for them.

Reply:
I wonder if we could get the article renderer to include the calls to action at the bottom of the article (in addition to where they are currently rendered). It also would be handy to get these to render on relevant samples.

One of the benefits of CTAs is that they are not checked for personal voice so we can directly address the reader in a way that the rest of the docs specifically prohibits. I wonder if there's a missing principle here.

Reply:
The checks for "personal voice" do limit the engagement we can put on these landing pages.
I wonder if there is some kind of compromise, some as the learning path pages, where it seems  "personal voice" is not enforced.